__tests__/app.test.js

```diff
@@ -168,3 +168,100 @@ describe("GET", () => {
       });
   });
 });
+
+describe("POST /api/reviews/:review_id/comments", () => {
+  it("should respond with username and body with comment", () => {
+    const commentInput = {
+      body: "I am slow but i am committed",
+      author: "mallionaire",
+    };
+    return request(app)
+      .post("/api/reviews/4/comments")
+      .send(commentInput)
+      .expect(201)
+      .then((response) => {
+        const { body } = response;
+        const { comment } = body;
+
+        expect(comment).toEqual({
+          comment_id: expect.any(Number),
```

Comment:
You marked Béla's comment as resolved, but this line doesn't seem to have changed. As said before, you should check the comment_id using the number of comments you know are already in the database from the test data 👍